lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -33,6 +35,21 @@ static int getTensorRank(Value tensor) {
   return tensorRank;
 }
 
+// Helper function to return the int_value for the given dtype.
+static int getIntForDtype(Type type) {
```

Comment:
There is a very similar helper in RefineTypes https://github.com/llvm/torch-mlir/blob/127c7d8e27ede211af77392b5277ba06fb515fb7/lib/Dialect/Torch/Transforms/RefineTypes.cpp#L35. Can you refactor the code and put the common part in Torch/Utils/Utils.h

Reply:
I think the better place would be `torch_upstream.h` since ScalarType enum is defined over there and this function also returns the same.

Reply:
torch_upstream.h are for things directly copied from upstream. We shouldn't include anything specific to our project there like #include "torch-mlir/Dialect/Torch/IR/TorchOps.h" or Type.

Reply:
Oh! I see thanks for the explanation.

Reply:
Done.